tests/test_collate_ab_transforms.py

```diff
@@ -0,0 +1,108 @@
+# ruff: noqa: PLR2004
+
+from pathlib import Path
+
+import pyarrow as pa
+import pytest
+
```

Comment:
Noting this here, as a little unsure where to note otherwise.  

Can we add `collate_ab_transforms` to the `abdiff.core.__init__` file, so it's directly importable?  Similar to how we've done other core functions?

Reply:
Updated.